### PR TITLE
Feature/emd 139 emd 140 site settings add site n layout

### PR DESCRIPTION
**Pull request by jmachnik**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -27,6 +27,7 @@ registerCallback("siteSettingsExtension", (ctx) => {
       <RouterView/>
     </main>
     <Toast/>
+    <ConfirmDialog></ConfirmDialog>
```
Comment (Conteusz81) [inline src/App.vue]: Why not ConfirmDialog /> ?

Comment (jmachnik) [reply src/App.vue]: no specific reason -> will change thx

**Review by Conteusz81 — DISMISSED** [review]

File: src/assets/_primereact.scss [hunk 2]
```diff
@@ -137,7 +140,10 @@ body {
   .p-datatable,
   .p-datatable.p-datatable-sm {
     .p-column-title {
+      line-height: 1.6;
       font-size: 14px;
+      font-weight: bold;
+      color: #343a40;
```
Comment (pbambynek) [inline src/assets/_primereact.scss]: isnt that some variable color? brand-gray something?

File: src/views/Sites.vue [hunk 3]
```diff
@@ -1,17 +1,58 @@
...
+    >
+      <Column selectionMode="multiple" headerStyle="width: 3em"></Column>
+      <Column field="code" header="Code" :show-filter-menu="false" sortable>
+        <template #filter="{ filterModel, filterCallback }">
+          <InputText
+              type="text"
+              v-model="filterModel.value"
+              @keydown.enter="filterCallback()"
+              class="p-column-filter"
+              :placeholder="`Search by name - `"
```
Comment (pbambynek) [inline src/views/Sites.vue]: why back-tick if no variable?

File: src/views/Sites.vue [hunk 4]
```diff
@@ -20,28 +61,70 @@
...
+            toast.add({
+              severity: "error",
+              summary: error.response.data.message,
+              detail: error.response.data.details[0].message,
+              life: 3000,
+            });
+          }
+        },
+        reject: () => {
+          //callback to execute when user rejects the action
```
Comment (pbambynek) [inline src/views/Sites.vue]: TO DO? or noop?

**Review by pbambynek — APPROVED** [review]

File: src/views/Site.vue [hunk 5]
```diff
@@ -25,25 +26,47 @@ export default {
       },
     ]);
     (async () => {
```
Comment (michalbujalski) [inline src/views/Site.vue]: This should be wrapped in `onMounted`

**Review by pbambynek — DISMISSED** [review]

File: src/views/Sites.vue [hunk 6]
```diff
@@ -20,28 +61,67 @@
...
+    const toast = useToast();
+    const selectedSites = ref();
     const sites = ref([]);
+    const {tenant} = useStore();
+    const confirm = useConfirm();
+    const filters = ref({
+      code: {value: null, matchMode: FilterMatchMode.CONTAINS},
+      name: {value: null, matchMode: FilterMatchMode.CONTAINS},
+    });
     (async () => {
```
Comment (michalbujalski) [inline src/views/Sites.vue]: wrap in onMount

**Review by michalbujalski — APPROVED** [review]

**Pull request merged**